integrations/sources/kafka.mdx

```diff
@@ -58,6 +58,11 @@ RisingWave performs primary key constraint checks on tables but not on sources.
 | group.id.prefix               | Optional. Specify a custom group ID prefix for the source. The default prefix is `rw-consumer`. Each job (materialized view) will have a separate consumer group with a generated suffix in the group ID, so the format of the consumer group is `{group_id_prefix}-{fragment_id}`. This is used to monitor progress in external Kafka tools and for authorization purposes. RisingWave does not rely on committed offsets or join the consumer group. It only reports offsets to the group. |
 | properties.sync.call.timeout  | Optional. Specify the timeout. By default, the timeout is 5 seconds.                                                                                                                                                                                                                                                                                                                                                                                                                        |
 | properties.client.id          | Optional. Client ID associated with the Kafka client.                                                                                                                                                                                                                                                                                                                                                                                                                                       |
+<Tip>
+**FAQ: GroupAuthorizationFailed**
+
+If you encounter `GroupAuthorizationFailed(Broker: Group authorization failed)` while consuming from Kafka topics, consider adding all consumer groups with the `group.id.prefix` (default: `rw-consumer`) to the ACL whitelist.
```

Comment:
Let's provide an example:
```
confluent kafka acl create \
    --allow \
    --service-account <service-account-id> \
    --operation READ \
    --consumer-group rw-*

confluent kafka acl list
```

And link service account docs: https://docs.risingwave.com/cloud/service-account#set-up-service-accounts-for-applications.

Reply:
I prefer putting the example command on the page dedicated to confluent.
This page is for more general purposes and maybe putting commands operating Apache Kafka is better here. 

```
./kafka-acls.sh --bootstrap-server <broker-addr> \
--add --allow-principal User:<user or service account> \
--consumer-group rw-consumer-.* \
--operation READ

```

Reply:
Okay we can do that in a separate PR, and link there.